demo.html

```diff
@@ -0,0 +1,21 @@
+<!doctype html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport"
+          content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <title>Document</title>
+    <script src="./dist/negotiator.js"></script>
+</head>
+<body>
+<script>
+    let n = new Negotiator();
+    n.authenticate({
+        unsignedToken: {},
+        unEndPoint: 'https://crypto-verify.herokuapp.com/use-devcon-ticket'
+    }).then(console.log);
+</script>
+
+</body>
+</html>
```

Comment:
Can we move this demo example of authenticate into the Hotel Bogota website.

Reply:
Feel free to move example. example just for you to display how it works.